### PR TITLE
[ADD] partner_label

### PR DESCRIPTION
**Pull request by hbrunn**
Partner labels
==============

This module was written to allow users to print address labels for partners.

Configuration
=============

To configure this module, you need to:

- go to `Settings / General settings`
- scroll to header `Labels configuration`
- change width, height, padding and margin to fit your label stickers, if necessary change the paper format
- you can preview your changes by pressing the `Preview` button

Usage
=====

- mark partners
- print your labels

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
Very cool module! One observation:
-Option to print a line in margins between labels, so that the labels are easier to cut

**Review by JordiBForgeFlow — APPROVED** [review]

**Review by lasley — APPROVED** [review]
Neato - thanks @hbrunn

File: partner_label/views/base_config_settings.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
```
Comment (lasley) [inline partner_label/views/base_config_settings.xml]: Please switch to `<odoo>` and remove data tag

**Comment by hbrunn** [comment]
@jbeficent thanks for bringing up invoices, I totally forgot about the contact widget, much better. This way, we profit from improvements to the contact widget

**Comment by JordiBForgeFlow** [comment]
Good!

**Comment by hbrunn** [comment]
about the cutting marks: I made this for example for those: https://www.avery.nl/artikel/adresetiketten-j8159-25 - does anyone really cut this manually?

**Comment by lasley** [comment]
> does anyone really cut this manually?

Seems like a lot of work IMO. And probably more cost by the time you factor in tape (assuming they are using regular paper) and labor.

**Comment by JordiBForgeFlow** [comment]
I was actually thinking of the cases when you print the labels on paper, when you want to put them on a plastic
image

**Comment by JordiBForgeFlow** [comment]
But anyway, that's a minor point.

**Comment by hbrunn** [comment]
to hack this, the user could define a paper format for the labels, and have the printer driver distribute the pages. cups allows to add cut marks easily

**Review by elicoidal — APPROVED** [review]
LGTM

**Pull request merged**